WordPress/Sniffs/Theme/NoCDNSniff.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Restricts the use of the CDN URLs.
+ *
+ * @link https://make.wordpress.org/themes/handbook/review/required/
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_NoCDNSniff extends WordPress_AbstractFunctionParameterSniff {
+
+	/**
+	 * The group name for this group of functions.
+	 *
+	 * @since 0.xx.0
+	 *
+	 * @var string
+	 */
+	protected $group_name = 'no_cdn';
+
+	/**
+	 * Array of functions with positions.
+	 *
+	 * The number represents the position in the function call
+	 * passed variables, here the capability is to be listed.
+	 *
+	 * @since 0.xx.0
+	 *
+	 * @var array
+	 */
+	protected $target_functions = array(
+		'wp_enqueue_style'  => 2,
+		'wp_enqueue_script' => 2,
+	);
+
+	/**
+	 * Array of CDN URLs.
```

Comment:
Is this an array of *whitelisted* or *blacklisted* CDNs ? Please expand the short description a little to make this clearer.

You may also want to consider changing the property name to `$cdn_url_blacklist` or just `$cdn_blacklist`.